error.go

```diff
@@ -33,6 +33,32 @@ func IsCannotReuseRelease(err error) bool {
 	return false
 }
 
+var (
+	emptyChartTemplatesRegexp = regexp.MustCompile(`release \S+ failed: no objects visited`)
```

Comment:
`no objects visited` is pretty cryptic. So I've named the error `emptyChartTemplates` as this is the only time we see this so far.